test/blackbox-tests/test-cases/include-dirs-include.t

```diff
@@ -0,0 +1,132 @@
+----------------------------------------------------------------------------------
+Tests for the `include` term in the (include_dirs ...) field of (foreign_stubs ...).
+
+  $ echo "(lang dune 3.5)" > dune-project
+
+----------------------------------------------------------------------------------
+* Refer to a single include dir with `include`:
+
+  $ cat >dune <<EOF
+  > (rule
+  >  (target incs)
+  >  (action
+  >   (with-stdout-to
+  >    incs
+  >    (run echo "(:standard inc_a)"))))
+  > (library
+  >  (name foo)
+  >  (foreign_stubs
+  >   (language c)
+  >   (names bar)
+  >   (include_dirs (include incs))))
+  > EOF
+
+  $ cat >bar.c <<EOF
+  > #include <caml/mlvalues.h>
+  > #include <a.h>
+  > value bar(value unit) { return Val_int(A); }
+  > EOF
+
+  $ mkdir -p inc_a
+
+  $ echo "#define A 42" > inc_a/a.h
+
+  $ dune build
+
+----------------------------------------------------------------------------------
+* Refer to multiple include dirs with a single `include` target:
+
+  $ cat >dune <<EOF
+  > (rule
+  >  (target incs)
+  >  (action
+  >   (with-stdout-to
+  >    incs
+  >    (run echo "(:standard inc_a inc_b)"))))
+  > (library
+  >  (name foo)
+  >  (foreign_stubs
+  >   (language c)
+  >   (names bar)
+  >   (include_dirs (include incs))))
+  > EOF
+
+  $ cat >bar.c <<EOF
+  > #include <caml/mlvalues.h>
+  > #include <a.h>
+  > #include <b.h>
+  > value bar(value unit) { return Val_int(A + B); }
+  > EOF
+
+  $ mkdir -p inc_a inc_b
+
+  $ echo "#define A 40" > inc_a/a.h
+  $ echo "#define B 2" > inc_b/b.h
+
+  $ dune build
+
+----------------------------------------------------------------------------------
+* Refer to zero include dirs with a single `include` target:
+
+  $ cat >dune <<EOF
+  > (rule
+  >  (target incs)
+  >  (action
+  >   (with-stdout-to
+  >    incs
+  >    (run echo "()"))))
+  > (library
+  >  (name foo)
+  >  (foreign_stubs
+  >   (language c)
+  >   (names bar)
+  >   (include_dirs (include incs))))
+  > EOF
+
+  $ cat >bar.c <<EOF
+  > #include <caml/mlvalues.h>
+  > value bar(value unit) { return Val_int(42); }
+  > EOF
+
+  $ dune build
+
+
+----------------------------------------------------------------------------------
+* Versions of dune before 3.5 do not support this feature
+
+  $ echo "(lang dune 3.4)" > dune-project
+
+  $ cat >dune <<EOF
+  > (rule
+  >  (target incs)
+  >  (action
+  >   (with-stdout-to
+  >    incs
+  >    (run echo "(:standard inc_a)"))))
+  > (library
+  >  (name foo)
+  >  (foreign_stubs
+  >   (language c)
+  >   (names bar)
+  >   (include_dirs (include incs))))
+  > EOF
+
+  $ cat >bar.c <<EOF
+  > #include <caml/mlvalues.h>
+  > #include <a.h>
+  > value bar(value unit) { return Val_int(A); }
+  > EOF
+
+  $ mkdir -p inc_a
+
+  $ echo "#define A 42" > inc_a/a.h
+
+  $ dune build
+  Warning: (include ...) in include_dirs field is only available since version
+  3.5 of the dune language. Please update your dune-project file to have (lang
+  dune 3.5).
```

Comment:
```suggestion
  Warning: (include ...) in ``include_dirs`` field is only available since version
  3.5 of the Dune language. Please update your ``dune-project`` file to have (lang
  dune 3.5).
```

Reply:
I've quoted "include_dirs". The rest of the suggested changes here are in text generated by `Syntax.Error_msg.since`.